app/views/forever_style_guide/sections/components/navigation/_nav_products_dropdown.erb

```diff
@@ -42,81 +42,70 @@
-              <small class="dropdown-menu-nav-link-subhead">Windows Scrapbooking Software</small>
-          </li>
-
-          <li>
-            <%= link_to 'Digital Art', digital_art_url, class: 'dropdown-menu-nav-link', title: 'Premium Digital Scrapbooking Art' %>
-            <small class="dropdown-menu-nav-link-subhead">Art Kits and Templates for Artisan</small>
+            <%= link_to trademark('Forever Print'), prints_url, class: 'dropdown-menu-nav-link', title: 'Create Premium Photo Books in No Time' %>
+            <small class='navbar-new_tag color-warning font-size-000 font-face-bold'>NEW</small>
+            <small class="dropdown-menu-nav-link-subhead">Online Photo Books</small>
 
             <ul class="list-unstyled dropdown-menu-nav-bullets">
               <li>
-                <%= link_to '- New Digital Art', new_digital_art_url, class: 'dropdown-menu-nav-link', title: 'Newest Digital Scrapbooking Art and Embellishments' %>
+                <%= link_to '- Featured Photo Books', featured_photo_books_url, class: 'dropdown-menu-nav-link', title: 'Featured Photo Books' %>
               </li>
 
               <li>
-                <%= link_to '- Free Kits', free_digital_art_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-no_margin', title: 'On Sale Digital Scrapbooking Art and Embellishments' %>
-              </li>
-
-              <li>
-                <%= link_to '- Browse All', all_digital_art_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-no_margin', title: 'All Digital Scrapbooking Art and Embellishments' %>
+                <%= link_to '- Browse All', all_photo_books_url, class: 'dropdown-menu-nav-link dropdown-menu-nav-link-no_margin', title: 'Browse All Photo Books' %>
               </li>
             </ul>
           </li>
 
-          <li class="hide">
-            <%= link_to 'Forever Projects', projects_marketing_url, class: 'dropdown-menu-nav-link', title: 'Make Photo Books and Other Print Projects' %>
-            <small class="dropdown-menu-nav-link-subhead">Online Photo Books and More</small>
-          </li>
-
           <li>
-            <%= link_to trademark('pixels2Pages'), p2p_url, class: 'dropdown-menu-nav-link', title: 'Premium Training and Resources' %>
-            <small class="dropdown-menu-nav-link-subhead">Premium Training and Resources</small>
+            <%= link_to trademark('Forever Artisan'), artisan_url, class: 'dropdown-menu-nav-link', title: 'Forever Artisan 5 Digital Scrapbooking Software' %>
+              <small class="dropdown-menu-nav-link-subhead">Windows Scrapbooking Software</small>
           </li>
 
+          <ul class="list-unstyled dropdown-menu-nav-bullets">
+            <li>
+              <%= link_to '- Photo Books, Cards, Calendars, & More', new_digital_art_url, class: 'dropdown-menu-nav-link', title: 'Artisan Photo Books, Cards, Calendars, & More' %>
```

Comment:
new_digital_art_url should not be here. I think the intent was to link to an anchor tag on the /artisan page. Can you confirm an id will be available to link to that way @amiballo

Reply:
yes was working on this now, I had not assigned yet cause I was ironing some of these details out, sorry for confusion prob could have waited to push up :)